orthoinference/src/main/java/org/reactome/orthoinference/StableIdentifierGenerator.java

```diff
@@ -52,16 +52,10 @@ public GKInstance generateOrthologousStableId(GKInstance inferredInst, GKInstanc
         // TODO: Performance check
         Collection<GKInstance> existingStableIdentifier = (Collection<GKInstance>) dba.fetchInstanceByAttribute("StableIdentifier", "identifier", "=", targetIdentifier);
 
-        GKInstance orthoStableIdentifierInst;
+        GKInstance orthoStableIdentifierInst = null;
```

Comment:
Just curious why the variable is initialized to null here.  Will the if/else always give it a value?  If so, it can be left just as a declaration.  Usually setting to null indicates that there's a possibility it will remain null and it had to be set so that the variable has a value when it gets used later.

Reply:
This was an IDE thing. Once I moved `orthoStableIdentifierInst` to an argument of the new method, it required it to be initialized in some way. ¯\_(ツ)_/¯

Reply:
Oh okay.  Actually, you should now be able to remove `orthoStableIdentifierInst` as an argument to the new method.  You can declare it locally in the method and return it.  That should also get rid of the IDE forcing the initialization to null.